Tests/FoundationTests/DateExtensionsTests.swift

```diff
@@ -957,6 +957,11 @@ final class DateExtensionsTests: XCTestCase {
         let date = Date()
         XCTAssertEqual(date.yesterday.timeIntervalSince(date), -86400.0)
     }
+
+    func testTomorrow() {
+        let date = Date()
+        XCTAssertTrue(date.tomorrow.isInTomorrow)
```

Comment:
For similar reasons to the ones discussed in https://github.com/SwifterSwift/SwifterSwift/pull/578 please use the same type of test as for `testYesterday`.